app/helpers/dateFormat.js

```diff
@@ -24,16 +24,11 @@ export function dateToUTC(timestamp) {
     date.getUTCHours(), date.getUTCMinutes(),  date.getUTCSeconds());
 }
 
-// endOfDay returns a new date pointing to the end of the day (last second)
-// of the day stored in the given date.
-export function endOfDay(dt) {
-  const res = new Date(dt);
-  res.setMilliseconds(0);
-  res.setSeconds(-1);
-  res.setMinutes(0);
-  res.setHours(0);
-  res.setDate(res.getDate()+1);
-  return res;
+// Returns a new date pointing to the last microsecond of the day of the given date
```

Comment:
Small nitpick: it's pointing to the last *milli*second instead of the last *micro*second.

Reply:
Good catch

Reply:
Fixed